snapshots/storage/bolt.go

```diff
@@ -366,18 +366,18 @@ func CommitActive(ctx context.Context, key, name string, usage snapshots.Usage,
 			return err
 		}
 		if err := bkt.DeleteBucket([]byte(key)); err != nil {
-			return errors.Wrap(err, "failed to delete active")
+			return errors.Wrapf(err, "failed to delete active snapshot %q", key)
 		}
 		if si.Parent != "" {
 			spbkt := bkt.Bucket([]byte(si.Parent))
 			if spbkt == nil {
-				return errors.Wrap(errdefs.ErrNotFound, "missing parent")
+				return errors.Wrapf(errdefs.ErrNotFound, "missing parent snapshot of %q", key)
```

Comment:
I think it would be good to include `si.Parent` in the error message as well:  `"missing parent %q of snapshot %s", si.Parent, key`